tasks/increase_timestamp.ts

```diff
@@ -0,0 +1,23 @@
+import { task, types } from 'hardhat/config'
```

Comment:
Hmm, not sure if we should add this task to the repo, even the sending ETH one above, and set_timestamp

Reply:
@kyriediculous already wrote that (some?) of these should be moved to scripts instead of tasks.